app/qml/components/MMNotification.qml

```diff
@@ -30,6 +29,8 @@ Rectangle {
     }
   }
 
+  Component.onCompleted: scale = 0.0
```

Comment:
Why is this important?

Reply:
I'd prefer not using `scale` altogether

Reply:
This is an initial value for the animation when notification starts